src/System.ServiceModel.Federation/src/System/ServiceModel/Federation/WsTrustChannelSecurityTokenManager.cs

```diff
@@ -21,13 +18,13 @@ public class WSTrustChannelSecurityTokenManager : ClientCredentialsSecurityToken
         private WSTrustChannelClientCredentials _wsTrustChannelClientCredentials;
 
         /// <summary>
-        ///
+        /// Instaintiates a <see cref="WSTrustChannelSecurityTokenManager"/>.
         /// </summary>
```

Comment:
Typo on Instantiates

Reply:
fixed in: b160df7f33caf6447f14e47b81ab3ff9f5930924

Reply:
fixed in: 9a8f277d85bf9419167870cad953b59d0d7a6559